### ISSUE TITLE
GOSW24- Email notification feature after submitting the contact us form.

### ISSUE THREAD

**Issue by nanda-mik** [GameofSource-GFG/Web-Development#55]
**Features to work on:**

1. Email should be sent to the lead when a particular person fills the contact us form with the details.
2. You can use express with nodemailer and sendgrid for this feature.
3. You can come up with other way for this feature and explain your approach while submitting the PR.

**Always note this:**

1. Use **comments** for better explanation of your code.
2. Try to resolve conflicts in the package.json files before submitting the PR.

**Comment by Priyanshu-official** [comment]
@nanda-mik I want to work on this assign to me pls

**Comment by geekyaditya100** [comment]
@Priyanshu-official is the gosw10 completed

**Comment by Priyanshu-official** [comment]
@geekyaditya100 Yes just a few changes i.e adding routing that's it and making PR in 10-15 min

**Comment by Priyanshu-official** [comment]
@geekyaditya100 Assign me, I have done my gosw10 just need review

**Comment by geekyaditya100** [comment]
@Priyanshu-official you can now work on this issue

**Comment by Priyanshu-official** [comment]
@geekyaditya100 Sorry I am Woking on this Pls give me a little more time I am busy Fixing the Conflict file 
I also have done with GOSW31  Footer Just need To make PR but Because of my recent PR I got an error when I make new PR
(Error -> non-fast-forward)
when GOSW10 Merged I will do Pr of others also
